### ISSUE TITLE
Support primary key personalize column name at VoyagerBreadController.php

### ISSUE THREAD

**Issue by MiguelOrdnz** [thedevdojo/voyager#667]
- Laravel Version: 5.3.28
- Voyager Version: 0.10.11
- PHP Version: 5.6.28
- Database Driver & Version: 
            Mysql  Ver 14.14 Distrib 5.7.16, for Linux (x86_64)
            Client API library version => mysqlnd 5.0.11-dev

### Description:
I got this error while trying to access a BREAD, i supossed that the issue was retaled to personalized primary key column.

`Column not found: 1054 Unknown column 'id' in 'order clause' (SQL: select * from `departamento` order by `id` desc)`

Then i track down till this class TCG\Voyager\Http\Controllers\VoyagerBreadController at line 44 where generate getting the collection ordered by "id" as default, i change this line:

`$dataTypeContent = call_user_func([$model->orderBy($model->getKeyName(), 'DESC'), $getter]);`

Please correct me if i'm wrong, if is not then i recomend this as a change since Eloquent supports customs primary key columns name Voyager should do it as well.

Regards.

**Comment by Hayat-Umar** [comment]
Exactly, I was just going to write about this issue here when i saw yours. Voyager should support custom Primary Key names. Firstly i was going to change id with {$model->getKeyName()} everywhere but i thought it will be good if TCG team do it themselves.

**Comment by marktopper** [comment]
I'm open for pull requests on this.

**Comment by gausam** [comment]
Got bit by this but on a table that's for continent lookups. It's an app I inherited in a table with just the continent_code and continent_name, so no timestamps or primary key.

The biter in the BREAD Controller is:

```php

$dataTypeContent = call_user_func([
    $model->with($relationships)->orderBy($model->getKeyName(), 'DESC'),
        $getter,
    ]);
```

Which by default returns `id`. Easy to work around per model though, but I wonder what an optimal solution would be Voyager-wise.

**Comment by fletch3555** [comment]
Voyager needs to remain agnostic around keys, which is why we use `getKeyName()`.  Do you have anything in the model identifying your table's primary key? What is the primary key, the `continent_code`?

**Comment by gausam** [comment]
@fletch3555 Yes, setting the primary key to `continent_code` in the model is how I resolved it for them.

Totally agree on the need for an agnostic way via `getKeyName()`. I was thinking more around what a better DevX would be, because the error floored a failry-new-to-Laravel dev which just made me think that there could be a more graceful way to fail than `Column not found: 1054 Unknown column 'id'`.

**Comment by handiwijoyo** [comment]
This issue should be fixed in `1.0`. Please open another issue if this still happen.
